wrappers/python/pyrs_pipeline.cpp

```diff
@@ -48,6 +48,9 @@ void init_pipeline(py::module &m) {
              "Upon calling resolve(), the config checks for conflicts between the application configuration requests and the attached computer vision "
              "modules and processing blocks requirements, and fails if conflicts are found.\n"
              "Before resolve() is called, no conflict check is done.", "stream_type"_a, "stream_index"_a, "width"_a, "height"_a, "format"_a, "framerate"_a)
+        
+        
+        .def("enable_stream", []( rs2::config* c, rs2_stream s ) -> void { return c->enable_stream( s, -1 ); }, "Stream type only. Other parameters are resolved internally.", "stream_type"_a )
```

Comment:
Why the explicit `-1` instead of using the C++ `enable_stream` that takes only 1 arg? (i.e., it has a default second arg)

Reply:
It can be yes, I wanted to show that I explicitly call this command arguments but without it it will also work the same.
Do you think it's preferred?

Reply:
The only thing I don't like is the explicit -1, because it assumes to know what the default is. Up to you how to do it.